itests/itest-include.bndrun

```diff
@@ -15,7 +15,7 @@ Test-Cases: ${classes;CONCRETE;PUBLIC;NAMED;*Test}
 -runsystempackages: sun.reflect
 
 -runfw: org.eclipse.osgi
--runee: JavaSE-11
```

Comment:
Can you also remove https://github.com/openhab/openhab-core/blob/3fc4d2373445c4df479c7f16dc450e5af847ff8d/itests/itest-include.bndrun#L39 ?